client/app/src/screens/ThankYou.js

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from 'react';
+import { Button, Text } from 'react-native-elements';
+import EStyleSheet from 'react-native-extended-stylesheet';
+import uuidv4 from 'uuid/v4';
+import { Clipboard, View } from 'react-native';
+
+import { MTURK_API_URL, MTURK_API_STUB, MTURK_APP_SECRET } from 'react-native-dotenv';
+import COLORS from '../constants/Colors';
+import WaitingScreen from './Waiting';
+import { store } from '../state';
+
+import { MESSAGE_THANK_YOU_HEADER, MESSAGE_THANK_YOU_BODY } from '../constants/Messages';
+
+export default class ThankYouScreen extends Component {
+  state = {
+    uuid: null,
+  };
+
+  componentWillMount() {
+    const id = uuidv4();
+    this.setState({ uuid: id });
+
+    const { songId, choiceType, choiceInverted, interval, timeout } = this.props.navigation.state;
+
+    const { demographics, choices } = store.getState();
+
+    const fetchParams = {
+      method: 'POST',
+      headers: {
+        authentication: 'Bearer ' + MTURK_APP_SECRET,
+      },
+      body: JSON.stringify({
+        id,
+        songId,
+        choiceType,
+        choiceInverted,
+        interval,
+        timeout,
+        demographics,
+        choices,
+      }),
+    };
+
+    if (MTURK_API_STUB === 'true') {
+      console.debug('FETCH params:', fetchParams);
+      return;
+    }
+
+    fetch(MTURK_API_URL, fetchParams)
+      .then(response => {
+        if (!response.ok) {
+          console.debug('FETCH response status:', response.status);
+        }
+      })
+      .catch(error => {
+        console.debug('FETCH error:', error);
+      });
```

Comment:
It would be more consistent with the rest of our app if this were done using async/await and try/catch:
```js
try {
  const response = await fetch(...);
  if (!response.ok) {
    throw response;
  }
} catch (error) {
  // Do stuff
}
```